zulipterminal/model.py

```diff
@@ -189,18 +189,21 @@ def _notify_server_of_presence(self) -> None:
                 }
             )
         self.new_user_input = False
+        return response
 
     @asynch
     def _start_presence_updates(self) -> None:
         """
-        Notify server of user's presence each minute (version 1a)
+        Use 'response' to update user list every minute (version 1b).
```

Comment:
This docstring change seems to reflect the added functionality, but you've lost some information; do you understand what?

Reply:
That I should add `Notify server of user's presence each minute` as a doc-string to the `_notify_server_of_presence` function ? :)